plugins/generator-datapack-1.17.x/datapack-1.17.x/utils/mcitems.ftl

```diff
@@ -56,115 +29,7 @@
-                    {"name": "east", "value": "none"},
-                    {"name": "north", "value": "none"},
-                    {"name": "south", "value": "none"},
-                    {"name": "up", "value": "true"},
-                    {"name": "waterlogged", "value": "false"},
-                    {"name": "west", "value": "none"}
-                    ] />
-                <#elseif ge.blockBase?has_content && ge.blockBase == "Leaves">
-                    <#assign properties += [
-                    {"name": "distance", "value": "7"},
-                    {"name": "persistent", "value": "false"}
-                    ] />
-                <#elseif ge.blockBase?has_content && ge.blockBase == "TrapDoor">
-                    <#assign properties += [
-                    {"name": "facing", "value": "north"},
-                    {"name": "half", "value": "bottom"},
-                    {"name": "open", "value": "false"},
-                    {"name": "powered", "value": "false"},
-                    {"name": "waterlogged", "value": "false"}
-                    ] />
-                <#elseif ge.blockBase?has_content && ge.blockBase == "Pane">
-                    <#assign properties += [
-                    {"name": "east", "value": "false"},
-                    {"name": "north", "value": "false"},
-                    {"name": "south", "value": "false"},
-                    {"name": "waterlogged", "value": "false"},
-                    {"name": "west", "value": "false"}
-                    ] />
-                <#elseif ge.blockBase?has_content && ge.blockBase == "Door">
-                    <#assign properties += [
-                    {"name": "facing", "value": "north"},
-                    {"name": "half", "value": "lower"},
-                    {"name": "hinge", "value": "left"},
-                    {"name": "open", "value": "false"},
-                    {"name": "powered", "value": "false"}
-                    ] />
-                <#elseif ge.blockBase?has_content && ge.blockBase == "FenceGate">
-                    <#assign properties += [
-                    {"name": "facing", "value": "north"},
-                    {"name": "in_wall", "value": "false"},
-                    {"name": "open", "value": "false"},
-                    {"name": "powered", "value": "false"}
-                    ] />
-                <#else>
-                    <#if ge.isWaterloggable>
-                        <#assign properties += [{"name": "waterlogged", "value": "false"}] />
-                    </#if>
-
-                    <#if ge.rotationMode != 0 && ge.rotationMode != 5>
-                        <#assign properties += [{"name": "facing", "value": "north"}] />
-                    <#elseif ge.rotationMode == 5>
-                        <#assign properties += [{"name": "facing", "value": "south"}] />
-                    </#if>
-                </#if>
-            </#if>
-
-            <#if properties?has_content>

```

Comment:
This part is 200% needed for dimensions to even remotely work

Reply:
But this code is inside a condition checking if the entry starts by `CUSTOM:` which is impossible to be true as custom blocks are not supported in data packs.

Reply:
You are right, I confused this with the part for defaultstates. All good :)